homeassistant/components/vizio/const.py

```diff
@@ -27,6 +26,7 @@
 )
 import homeassistant.helpers.config_validation as cv
 
+SERVICE_FETCH_LATEST_APPS = "fetch_latest_apps"
```

Comment:
Should we really make this a service? Why can't we just fetch it every couple of hours? What are the odds someone knows about and wants to use an app that was published recently?

Reply:
My original thought was to make it so the user had to choose to introduce an external dependency, but I can easily automate this.